src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -0,0 +1,52 @@
+package org.openmrs.contrib.qaframework.page;
+
+import java.util.Scanner;
+
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page {
+	
+	private static final By ATTACHMENTS_LINK = By.cssSelector("i.right:nth-child(1)");
+	private static final By ATTACH_FILE = By.cssSelector(".dz-default");
+	private static final By ADD_CAPTION = By.xpath("/html/body/div/div[3]/div[3]/div[1]/att-file-upload/div[2]/div/div[2]/textarea");
```

Comment:
try improving this reference, figure out a more less tree based access to it

Reply:
> try improving this reference, figure out a more less tree based access to it

This sounds simple. Not sure how but am going to find out.